.circleci/config.yml

```diff
@@ -594,6 +594,32 @@ jobs:
             go vet ./...
       - magma_slack_notify
 
+  feg-lint:
```

Comment:
You need to add `feg-lint` in the `feg` workflow to trigger it.

https://github.com/magma/magma/blob/master/.circleci/config.yml#L1146 <-- there

Reply:
yep i forgot !